platforms-serverless/firebase-functions/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "firebase-functions",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "devDependencies": {
+    "prisma": "4.13.0-dev.19"
+  },
+  "dependencies": {
+    "@prisma/client": "4.13.0-dev.19"
+  }
+}
```

Comment:
Necessary although package not even used here, but just in "functions"?

Reply:
Same as https://github.com/prisma/ecosystem-tests/pull/3463#discussion_r1160128518